Failed build: CI - with Numpy&#x2F;Scipy nightly wheels (nightly)

GitHub Actions workflow [CI - with Numpy&#x2F;Scipy nightly wheels (nightly) #464](https://github.com/jax-ml/jax/actions/runs/14945118704) failed.

Event: schedule
Branch: [main](https://github.com/jax-ml/jax/tree/main)
Commit: [35e2657be8308917c7fa407be5a0b53192134890](https://github.com/jax-ml/jax/commit/35e2657be8308917c7fa407be5a0b53192134890)

<sup><i>Created by [jayqi/failed-build-issue-action](https://github.com/jayqi/failed-build-issue-action)</i></sup>

The failures here look like this:
```
__________________ ERROR collecting tests/shape_poly_test.py ___________________
ImportError while importing test module '/__w/jax/jax/tests/shape_poly_test.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/shape_poly_test.py:58: in <module>
    from jax._src.internal_test_util import test_harnesses
E   ModuleNotFoundError: No module named 'jax._src.internal_test_util'
```
I suspect this is probably related to https://github.com/jax-ml/jax/pull/28650

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -88,7 +88,7 @@ doctest_optionflags = [
     "NUMBER",
     "NORMALIZE_WHITESPACE"
 ]
-addopts = "--doctest-glob='*.rst' --ignore='examples/ffi' --import-mode=importlib"
+addopts = "--doctest-glob='*.rst' --ignore='examples/ffi'"
 
 [tool.ruff]
 preview = true

